index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
```

Comment:
why this file is inside root ? not in public/ as should be

Reply:
that's vite requirements